doc/installation.rst

```diff
@@ -1,129 +1,217 @@
 
-After installation, open a new terminal and run ``iot`` from anywhere. (Or, for debugging, run ``bash run`` in the iot directory.)
+After the installation is finished, open a new terminal, and just run ``iot`` from anywhere
+(you can also run ``bash run`` in the iot directory if things go wrong,
+but only for debugging purposes).
+
+After successfully entering IoTempower (the prompt
+should have changed and started now with IoT),
+start configuring your first IoT node,
+see `First IoT Node <first-node.rst>`_.
+
 
-After entering IoTempower for the first time (prompt should now indicate IoT), start configuring your first IoT node (see `First IoT Node <first-node.rst>`_).
+Windows Host with WSL2 + usbipd-win
+-----------------------------------
+
+.. note::
+   **New Option:** For Windows 10 (2004+) and Windows 11 users, WSL2 combined
+   with ``usbipd-win`` provides a viable alternative to a full Linux installation.
+   This setup allows USB serial device access (for ESP32, ESP8266, etc.) from 
+   within WSL2, enabling full flashing and device communication.
 
-Windows Host via WSL2 + usbipd-win (Recommended)
-------------------------------------------------
+**Step 1: Install WSL2 and Ubuntu (or Debian)**
 
-*For Windows 11 and recent Windows 10, this is now the most robust and least-friction method.*
+In a Windows PowerShell (run as Administrator):
 
-1. Install WSL2 and Ubuntu (or Debian):
-   
 .. code-block:: powershell
-   
+
    wsl --install
 
-   # Restart as instructed, then in Ubuntu:
+Restart as instructed. Then open Ubuntu from the Start menu and run:
+
+.. code-block:: bash
+
    sudo apt-get update
    sudo apt-get install git curl
    curl -L https://now.iotempower.us | bash -
 
-2. Enable USB serial device access (ESP32/ESP8266 etc):
-   - Ensure you have Windows "App Installer" (winget) available (pre-installed on Windows 11, Win10 2004+).
-   - In a Windows command prompt or PowerShell (not WSL):
+**Step 2: Enable USB serial device access**
+
+To access USB devices (like ESP32/ESP8266) from WSL2, install ``usbipd-win``
+on the Windows host using ``winget``. ``winget`` is pre-installed on Windows 11
+and Windows 10 version 2004 and later. If unavailable, update "App Installer"
+from the Microsoft Store.
+
+In a Windows command prompt or PowerShell (**not** WSL):
 
 .. code-block:: powershell
 
    winget install --interactive --exact dorssel.usbipd-win
 
-   # Plug in your device, then:
+After installation, plug in your device and run:
+
+.. code-block:: powershell
+
    usbipd list
    usbipd attach --busid <BUSID_FROM_LIST>
 
-   # Your device will now appear under /dev/ttyUSB0 or similar in WSL2
+Your device will now appear under ``/dev/ttyUSB0`` or similar in WSL2.
+Use IoTempower as normal from your WSL2 Linux shell.
 
-   - Use as normal from WSL2 Linux shell. Full flashing and device access is now supported.
-   - If winget is not available, update "App Installer" from Microsoft Store.
 
 Docker/Podman (Deprecated)
```

Comment:
Docker is deprecated, Podman isn't. It is the recommendation if containers should be used - in linux, windows, or on mac.